.snyk

```diff
@@ -5,5 +5,5 @@ ignore:
   SNYK-JAVA-IONETTY-1042268:
     - '*':
         reason: No replacement available
-        expires: 2022-04-30T00:00:00.000Z
+        expires: 2022-05-30T00:00:00.000Z
```

Comment:
Isn't this vulnerability fixed with the upgrade?

Reply:
The initial issue was the expiration one, the other one was due to netty issue.
Upgraded the version, everything fine now

Reply:
Removing this now